packages/kbn-config-schema/src/types/object_type.test.ts

```diff
@@ -199,3 +199,19 @@ test('includes namespace in failure when wrong value type', () => {
 
   expect(() => type.validate(value, {}, 'foo-namespace')).toThrowErrorMatchingSnapshot();
 });
+
+test('reach returns a valid schema component', () => {
+  const type = schema.object({
+    foo: schema.boolean(),
+  });
+
+  const fooSchema = type.reach('foo');
+  const value = false;
+  expect(() => fooSchema.validate(value)).toBeTruthy();
+
+  try {
```

Comment:
`expect(() => type.reach('bar')).toThrowErrorMatchingInlineSnapshot()` will fail if `type.reach()` returns without throwing, but the current test will pass even if it doesn't throw. (run `node scripts/jest -u packages/kbn-config-schema/src/types/object_type.test.ts` to update the snapshot